src/AutomatedCar/ViewModels/MainWindowViewModel.cs

```diff
@@ -33,5 +33,7 @@ public ViewModelBase Dashboard
             get => this.dashboard;
             private set => this.RaiseAndSetIfChanged(ref this.dashboard, value);
         }
+
+
```

Comment:
Ezeket ki kellene törölni hogy az egyik automata codecheck ne dobjon hibát

Reply:
Igen, köszi közben én is megtaláltam meg pár tesztet is át kellett írni.